spec/support/products_for_users.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+shared_context 'products_for_users' do
+  before(:each) do
+    Rails.application.routes.draw do
+      devise_for :dummy_users
+      devise_scope :dummy_user do
+        products_for :dummy_users, resource_class_name: 'Flatirons::Saas::Product'
+      end
+    end
+  end
+end
```

Comment:
Maybe you can put this on routes.rb of dummy project like I did to subscriptions